stacks_queues/stack.py

```diff
@@ -21,18 +21,30 @@ def pop(self):
             The Stack is empty.
             returns None
         """
-        pass
+        if self.store == None:
+            raise StackEmptyException("Stack is empty!")
+
+        removed = self.store.get_first()    
+        self.store.remove_first()
+
+        return removed
```

Comment:
👀 LinkedList's `remove_first` returns the value it removes!
```suggestion
        return self.remove_first()
```